src/core/qobj.jl

```diff
@@ -1675,8 +1675,6 @@ fock_dm(i::Int64, hspace_size::Int64) = ket2dm(fock(i, hspace_size))
 Computes the Wigner function of the density matrix `ρ` at the point (`p`,`q`).
 
 ```jldoctest
-julia> using Printf
```

Comment:
This is where we need Printf. It seemed cleaner to import it in DocSetup

Reply:
We can just replace the `printf` macro with

`println("prob: $(round(prob; digits=6))")`

I don't think it's worth pulling in a dependency for just that line.

Reply:
Since the above produces

![image](https://github.com/SnowflurrySDK/Snowflurry.jl/assets/99685336/142202c7-93ce-4fca-a2e7-259c6f379d4e)

It will just be easier to keep it as it is